demo/paging/paging-virtual.component.ts

```diff
@@ -3,7 +3,7 @@ import {MockServerResultsService} from "./mock-server-results-service";
 import {PagedData} from "./model/paged-data";
 import {CorporateEmployee} from "./model/corporate-employee";
 import {Page} from "./model/page";
-
+//[columns]="[{name:'Name'},{name:'Gender'},{name:'Company'}]"
```

Comment:
Every change on demo/paging/paging-virtual.component.ts should have not been pushed through. I started using this demo to try out row grouping functionality before I actually created the exclusive demo.